README.md

```diff
@@ -263,6 +263,29 @@ usage: example [--name NAME]
 error: error processing --name: missing period in "oops"
 ```
 
+### Description strings
+
+```go
+type args struct {
+	...
+}
+
+func (args) Description() string {
+	return "this program does this and that"
+}
+
+func main() {
+	var args args
+	arg.MustParse(&args)
+}
+```
+
+```shell
+$ ./example -h
+this program does this and that
```

Comment:
can you add one field to the `args` struct to show users that description appears above the flags, and that it does not override the flags text?